.yamato/build_windows.yml

```
@@ -15,6 +15,12 @@ commands:
   - powershell .yamato\scripts\download_7z.ps1
   - artifacts\7za-win-x64\7za.exe a artifacts\unity\dotnet-runtime-unity-win-x86.7z .\artifacts\bin\microsoft.netcore.app.runtime.win-x86\Release\runtimes\win-x86
```

```suggestion
  - artifacts\7za-win-x64\7za.exe a artifacts\unity\dotnet-runtime-unity-win-x86.7z .\artifacts\bin\microsoft.netcore.app.runtime.win-x86\Release\runtimes\win-x86\*
```
Missed that in my first pass.

---

Done